announcements/_posts/2019-09-06-w02-everthing-ssh.md

```diff
@@ -0,0 +1,20 @@
+---
+author: Jeffery Russell (@jrtechs)
+title: "Week #2: Everything SSH!"
+layout: post-event
+date-start: "2019-09-06 16:30"
+date-end: "2019-09-06 18:00"
+location: "GOL-2620 (Med. DB Lab)"
+---
+
+Hey Ritluggers,
+
+This week we got a very excitting meeting: we are diving deep into SSH.
```

Comment:
* First phrase before the colon is phrased awkwardly. Either "This week we have...", or another variation. "This week we got..." isn't valid English in this context :P
* excitting(sp).